libs/libvtrutil/src/vtr_geometry.h

```diff
@@ -101,9 +102,17 @@ class Rect {
     //Returns true if the point is coincident with the rectangle (including the top-right edges)
     bool coincident(Point<T> point) const;
 
+    //Returns true if no points are contained in the rectangle
+    bool empty() const;
```

Comment:
Likely want to clarify this comment. Its a little unclear what 'no points are contained' means. I think it means the Rect has zero area (i.e. width or height = 0).

Reply:
It means `rect.empty() => forall p. rect.contains(p) == false`. I'll add this property to the comments, but it probably wouldn't make a good unit test.

Reply:
Or closer to the comment:  `rect.empty() => not exists p. rect.contains(p)`

Reply:
Done.